.github/workflows/Operator-CI.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Build Shard operator image and resources
         uses: ./.github/actions/maven
         with:
-          maven-command: clean install --errors --quiet -Pminikube -Dquarkus.container-image.tag=$GITHUB_SHA -Dnamespace=$NAMESPACE -Dquickly
+          maven-command: clean install --errors --quiet -Pminikube -Dquarkus.container-image.tag=$GITHUB_SHA -Dnamespace=$NAMESPACE -Dquickly -Dquarkus.kubernetes.namespace=$NAMESPACE
```

Comment:
I still don't get how the Operator-CI is related to the manage-tenant onboarding, could you clarify?

Reply:
Expectation
For manage-tenant on-boarding, generated operator-bundle should be independent of namespaces. Namespace value will be injected while creating addon.

Actual
With the current implementation, namespace attribute always injected in to Deployment and ServiceAccount while compiling project using maven.

Solution
Remove hardcoded default namespace reference from pom.xml and only provide namespace value while compiling project using
-Dquarkus.kubernetes.namespace=$NAMESPACE argument.